recipes/arm_pl.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: arm_pl
```

Comment:
I'd call the recipe: arm_pl_install.rb, in this way it's easier to distinguish recipes executed at install and config time.

Reply:
done